Cargo.lock

```diff
@@ -4072,8 +4070,7 @@ checksum = "957e51f3646910546462e67d5f7599b9e4fb8acdd304b087a6494730f9eebf04"
 [[package]]
 name = "unique_port"
 version = "0.2.1"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "d3f903132efabcb4242e0bbf16aedb2c3867526e42126b6ed3cb1b0bc4577708"
+source = "git+https://github.com/ales-tsurko/iroha2-unique_port?rev=09e95bb3608626c995b8e7e51c64ba7486c88948#09e95bb3608626c995b8e7e51c64ba7486c88948"
```

Comment:
I think this should point to https://github.com/soramitsu/iroha2-unique_port, not the fork

Reply:
From [this experience](https://github.com/hyperledger/iroha/pull/2363/files#r906053569) I don't think we need to take care of `Cargo.lock`. I guess @ales-tsurko 's work is now ahead of what crates.io has, thus this is temporary and acceptable, right?

Reply:
it seems like Ales's change was already merged upstream though

Reply:
IDK the exact reason, let's make it the same as in [the head branch](https://github.com/hyperledger/iroha/blob/5ff6b67d392a0f8238d483918ea36e82d9a22937/Cargo.lock#L4075)

Reply:
I'd leave as is. It will be automatically overwritten if necessary. We should not manually touch `Cargo.lock`